bftengine/src/bcstatetransfer/Messages.hpp

```diff
@@ -152,7 +156,7 @@ struct CheckpointSummaryMsg : public BCStateTranBaseMsg {
   }
 
   static bool equivalent(const CheckpointSummaryMsg* a, uint16_t a_id, const CheckpointSummaryMsg* b, uint16_t b_id) {
-    static_assert((sizeof(CheckpointSummaryMsg) - sizeof(requestMsgSeqNum) == 87),
+    static_assert((sizeof(CheckpointSummaryMsg) - sizeof(requestMsgSeqNum) == 88),
```

Comment:
why use a number in code? define a CONST

Reply:
That's how code was written, @hshomroni  nnly changed the number. I'm not sure we need it as a constexpr, since it is used only here. What's the point to define it one line above if it is used only here? I dont see a reason.